stripe-node/app.js

```diff
@@ -0,0 +1,58 @@
+// API with basic form
+// Set local environment variables
+const keyPublishable = 'pk_test';
+const keySecret = 'sk_test';
+// Set up express
+const app = require("express")();
+const stripe = require("stripe")(keySecret);
+// Todo: Clean up pug form, replace with angular
+app.set("view engine", "pug");
+var bodyParser = require('body-parser')
+// Send header to allow post request
+app.use(bodyParser.json());
+app.use(function(req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
```

Comment:
Any reason why we are allowing requests from anywhere?

Reply:
Hi Mark, I ran into a blocker restricting requests locally so this is for local development only. I'll add a note to make sure its removed.